galactic/release-rhel-build.yaml

```diff
@@ -71,40 +56,32 @@ package_blacklist:
 - octomap  # libqglviewer-dev-qt5 has no RPM for RHEL
 - ompl  # opende has no RPM package for RHEL
 - pcl_ros  # pcl RPM has no visualization component for RHEL
-- rc_dynamics_api  # Not yet generated for RHEL
-- rc_genicam_api  # Not yet generated for RHEL
 - rc_genicam_driver  # Not yet generated for RHEL
-- robot_localization  # Not yet generated for RHEL
+- robot_localization  # Build failure
```

Comment:
For this one, it actually looks like the source build is failing: https://build.ros2.org/view/Gbin_rhel_el864/job/Gsrc_el8__robot_localization__rhel_8__source/6/console .  Likely that was skipped during bloom generation, which further means that it is likely missing a dependency.

Reply:
You're right - I must have crossed the streams here. This package was missing a boost rule that has since been fixed. I reverted this line in 04c8a91.